project/MimaExcludes.scala

```diff
@@ -833,6 +833,11 @@ object MimaExcludes {
       // [SPARK-12221] Add CPU time to metrics
       ProblemFilters.exclude[DirectMissingMethodProblem]("org.apache.spark.status.api.v1.TaskMetrics.this"),
       ProblemFilters.exclude[DirectMissingMethodProblem]("org.apache.spark.status.api.v1.TaskMetricDistributions.this")
+    ) ++ Seq(
```

Comment:
They should be in `lazy val v21excludes = v20excludes`. Master is supposed to be released as 2.1.